cmd/crio/main.go

```diff
@@ -308,50 +320,60 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "pause-command",
+			Value: defConf.PauseCommand,
 			Usage: "name of the pause command in the pause image",
 		},
 		cli.StringFlag{
 			Name:  "pause-image",
+			Value: defConf.PauseImage,
 			Usage: "name of the pause image",
 		},
 		cli.StringFlag{
 			Name:  "pause-image-auth-file",
+			Value: defConf.PauseImageAuthFile,
 			Usage: "path to a config file containing credentials for --pause-image",
 		},
 		cli.StringFlag{
 			Name:  "signature-policy",
+			Value: defConf.SignaturePolicyPath,
 			Usage: "path to signature policy file",
 		},
 		cli.StringFlag{
 			Name:  "root",
+			Value: defConf.Root,
 			Usage: "crio root dir",
 		},
 		cli.StringFlag{
 			Name:  "runroot",
+			Value: defConf.RunRoot,
 			Usage: "crio state dir",
 		},
 		cli.StringFlag{
 			Name:  "storage-driver",
+			Value: defConf.Storage,
 			Usage: "storage driver",
 		},
 		cli.StringSliceFlag{
 			Name:  "storage-opt",
+			Value: toStringSlice(defConf.StorageOptions),
 			Usage: "storage driver option",
 		},
 		cli.BoolFlag{
 			Name:  "file-locking",
-			Usage: "enable or disable file-based locking",
+			Usage: fmt.Sprintf("enable or disable file-based locking (default: %t)", defConf.FileLocking),
```

Comment:
Was not using Value here intentional?

Reply:
Yes you might be right. How can we verify test this?